### DISCUSSION TITLE
Disable shadows for a particular "window"

### DISCUSSION THREAD

**Discussion by typecasto** [neovide/neovide#2630] [Q&A]
When saving a file (or for certain LSP features), this window will pop up along the bottom:
![image](https://github.com/neovide/neovide/assets/9502466/5216445d-1253-454c-a649-58378ed40a9a)

Can I turn off shadows for this "window"?

**Comment by xzbdmw** [comment]
You can see information in this pr.
https://github.com/neovide/neovide/pull/2408
Though it is not merged, I patch myself and can't live without it.
